applications/ws-affiliation-rnsr/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM inistcnrs/lodex-workers-python:4.0.11
+RUN pip install dvc[all]==2.48 fasttext==0.9.2
+WORKDIR /app/public
+COPY /models /app/public/
```

```suggestion
COPY ./public/ /app/public/
```

That way, source code *and* models `.dvc` will be copied (provided that we first put code and models directory inside `./public`).